src/rofi_rbw/selector.py

```diff
@@ -140,8 +140,8 @@ def show_selection(
             capture_output=True,
             encoding='utf-8'
         )
-        returnaction = None
         if wofi.returncode == 0:
-            returnaction = default_action
-        return returnaction, wofi.stdout
+            return default_action, wofi.stdout
+        else
+            return None, wofi.stdout
```

Comment:
We should make it return an optional `str`, because `CANCEL` doesn't have one.

Reply:
I think keeping it as is is easier to understand, as the stdout is still valid even if the user decides to cancel right?

Reply:
I don't think there is stdout if you cancel

Reply:
Yea there is none, but I mean that the stdout value can still be retrieved from the selector

Reply:
Yep, but it doesn't make sense - if the user cancelled, what is there to retrieve? Ideally, we could combine it so that only some actions have a value, and some don't.

Reply:
Hmm yea that'd be the cleanest solution, but I'm not sure if it's possible to add data in a clean and easy way.

Reply:
I don't think enums can have fields in Python (like `Action.COPY('👨‍🚀')`, which would be nice here. Another solution would be `Union[Tuple[Action, str], SelectorResponse]`, but that is kinda mean for the caller ;)

Probably an actual type that can be initialized either with `Action` and `str` or with `SelectorResponse` would be the best solution.